drivers/gpio/Kconfig.mchp_mss

```diff
@@ -0,0 +1,10 @@
+# Microchip PolarFire SoC Icicle Kit GPIO configuration options
+
+# Copyright (c) 2022 Microchip Technology Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+config GPIO_MCHP_MSS
+	bool "Microchip PolarFire SoC GPIO driver"
+	depends on SOC_SERIES_RISCV64_MIV
```

Comment:
remove `SOC_SERIES_RISCV64_MIV` and add:

```
default y
depends on DT_HAS_MICROCHIP_MPFS_GPIO_ENABLED
```

This utilizes an auto-generated Kconfig.dts file that will have the DT_HAS_MICROCHIP_MPFS_GPIO_ENABLED symbol defined.

Reply:
done